Changelog.md

```diff
@@ -8,6 +8,8 @@ Compiler Features:
  * SMTChecker: Support ``selector`` for expressions with value known at compile-time.
  * Command Line Interface: New option ``--model-checker-timeout`` sets a timeout in milliseconds for each individual query performed by the SMTChecker.
  * Standard JSON: New option ``modelCheckerSettings.timeout`` sets a timeout in milliseconds for each individual query performed by the SMTChecker.
+ * Commandline Interface: Perform linking in assembly mode when library addresses are provided.
+ * Standard JSON: Perform linking in assembly mode when library addresses are provided.
```

Comment:
Should I be adding two entries when the change affects both CLI and Standard JSON?

Reply:
What about `Assembler: ...`? It's not really a change to the commandline interface

Reply:
You have a point :) `Assembler` sounds good.

Reply:
Done.